src/Form/CoyoteImgDescForm.php

```diff
@@ -113,7 +146,7 @@ private function verifyResourceGroup(
         Util::getResourceGroupUri()
       );
       if (is_null($group)) {
-          $form['api_organization']['#field_suffix'] = $this->t("Resource group '@resourceGroup' could not be created", ['@resourceGroup' => Constants::RESOURCE_GROUP_NAME]);
+          $form['api_organization']['#field_suffix'] = $this->t("Resource group '@resourceGroup' could not be created", array('@resourceGroup' => Constants::RESOURCE_GROUP_NAME));
```

Comment:
Is the `array()` function over literal notation required for something?

Reply:
nope, personally I think array() is better readable, but opinions differ:
https://befused.com/php/short-array/

shall we use short array syntax everywhere?

Reply:
Yes please, I generally use this syntax across code, so if only for consistency's sake 👍